src/commands/Owner/Reload.js

```diff
@@ -32,7 +32,7 @@ class Reload extends Command {
     const commandsAddedString = commandsAdded.length > 0 ? commandsAdded.sort().join(' ') : ' No Commands Added';
     const commandsRemovedString = commandsRemoved.length > 0 ? commandsRemoved.sort().join(' ') : ' No Commands Removed';
 
-    this.messageManager.sendMessage(message, `${this.md.codeMulti}Commands reloaded!${this.md.blockEnd}` +
+    await this.messageManager.sendMessage(message, `${this.md.codeMulti}Commands reloaded!${this.md.blockEnd}` +
```

Comment:
we probably want to look at returning the values from the operations in the messageManager functions so that these and successes make more sense.